administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -29,10 +29,14 @@ const ApplicationList = styled(FlipMove)`
 export type Application = GetApplicationsQuery['applications'][number]
 
 // Necessary for FlipMove, as it cannot handle functional components
-export class ApplicationViewComponent extends React.Component<React.ComponentProps<typeof ApplicationCard>> {
+// eslint-disable-next-line react/prefer-stateless-function
+export class ApplicationViewComponent extends React.Component<ApplicationCardProps> {
   render(): ReactElement {
+    // eslint-disable-next-line react/prop-types
+    const { application } = this.props
     return (
-      <ApplicationListCard key={this.props.application.id}>
+      // eslint-disable-next-line react/prop-types
```

Comment:
Can't we fullfill the rules?

Reply:
I think not possible for class components without installing deprecated prop-types again :disappointed: I did not spend too much time investigating though.